polaris-react/src/components/Button/Button.tsx

```diff
@@ -144,29 +144,50 @@ export function Button({
       <div
         className={classNames(styles.DisclosureIcon, loading && styles.hidden)}
       >
-        <Icon
-          source={
-            loading
-              ? 'placeholder'
-              : getDisclosureIconSource(
-                  disclosure,
-                  ChevronUpMinor,
-                  ChevronDownMinor,
-                )
-          }
-        />
+        {loading ? (
+          <div
+            style={{
+              display: 'block',
+              height: '20px',
+              width: '20px',
+              maxHeight: '100%',
+              maxWidth: '100%',
+              margin: 'auto',
+              paddingBottom: '100%',
+              background: 'currentColor',
+            }}
+          />
```

Comment:
Note: This instance and the one below were the only actual usages of `Icon source="placholder"` in `polaris-react`. Therefore, I've opted to hard code the styled `div` equivalent instead of introducing a `PlaceholderMajor` Polaris Icon. Thoughts?